salt/states/virt.py

```diff
@@ -788,6 +789,7 @@ def pool_running(name,
                                                  permissions=permissions,
                                                  source_devices=(source or {}).get('devices', None),
                                                  source_dir=(source or {}).get('dir', None),
+                                                 source_initiator=(source or {}).get('initiator', None),
```

Comment:
FYI `.get()` already return `None` if the element is missing, can save a lot of boilerplate.

```
>>> {}.get('key') is None
True
```

Reply:
oh right